recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0-3" %}
+
+package:
+  name: r-gsw
```

Comment:
Actually, I am glad to hear that, because I also did not particularly like using these many variables. I will adapt the skeleton command accordingly.